backend/core/config.py

```diff
@@ -25,6 +27,19 @@
 app.include_router(chat_routes.router, tags=["chat"])
 
 
+@app.on_event("startup")
+async def startup_event():
+    await rabbitmq_client.initialize()
+    loop = asyncio.get_running_loop()
+    loop.create_task(rabbitmq_client.consumer())
+
+
+@app.post("/test-rabbitmq-send-message", tags=["dev"])
+async def send_message(message: str):
+    await rabbitmq_client.producer(message)
+    return {"status": "Message sent"}
+
+
 @app.get("/health-check", tags=["dev"])
```

Comment:
Should health checks not be a production feature? Does Kubernetes not use these? Ping @anjarakvo ?